src/nni_manager/common/manager.ts

```diff
@@ -103,6 +103,7 @@ abstract class Manager {
     public abstract exportData(): Promise<string>;
 
     public abstract addCustomizedTrialJob(hyperParams: string): Promise<void>;
+    public abstract resubmitTrialJob(hyperParams: string): Promise<void>;
     public abstract cancelTrialJobByUser(trialJobId: string): Promise<void>;
```

Comment:
the parameter should be trial job id as we discussed.

Reply:
fixed